src/components/widget/slideshow.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import CarSlides from './carSlides'
+
+class Slideshow extends React.Component{
+    constructor(props) {
+        super(props);
+        this.state = {curr_slide: CarSlides};
+    }
```

Comment:
Can you add a render method that calls on <CarSlides />, or at least a TODO if you cannot figure it out?